templates/corporate/support_request_thanks.html

```diff
@@ -0,0 +1,18 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="register-account flex full-page">
+    <div class="center-block new-style">
+        <div class="pitch">
+            <h1>Thanks for getting in touch with us!</h1>
+            <p>Your message has been successfully submitted.</p>
+            <p>
+                You can find answers to frequently asked questions in the
+                <a href="/help/">Zulip help center</a> or
+                <a href="/developer-community/">The Zulip developer community</a>.
```

Comment:
Hm, not sure about the text here. If we want to mention the developer community, I think we need to provide more guidance on when folks should go there. It's good for "I have a bug to report," but not for a variety of org-specific or user-specific requests.

Might be simplest to just link to the help center?

Reply:
Yeah, I'd just link the help center.